scrapyjs/middleware.py

```diff
@@ -32,6 +33,14 @@ def __init__(self, crawler, splash_base_url, slot_policy):
         self.splash_base_url = splash_base_url
         self.slot_policy = slot_policy
 
+    def get_splash_options(self, request, spider):
+        if request.meta.get("dont_proxy"):
```

Comment:
after sleeping on it I think it's bad idea to reuse 'dont_proxy' key here, because it's unexpected for developers who is using splash and crawlera middleware together in the project - they would expect it to work only with crawlera mw, so they can enable crawlera for spider (for all requests) and for some specific requests they might want to disable crawlera using 'dont_proxy' and add 'splash' arguments. 

I think explicit 'dont_splash' would be better here.